eahub/localgroups/models.py

```diff
@@ -121,6 +121,16 @@ def convert_to_row(self, field_names):
                 values.append(getattr(self, field))
         return values
 
+    @staticmethod
+    def get_exportable_field_names():
+        fieldnames = [
+            field.name
+            for field in LocalGroup._meta.fields + LocalGroup._meta.many_to_many
+            if field.name != "local_group_type"
+        ]
```

Comment:
looks tricky :)

although I don't have anything against this code really, I'm just not 100% sure re what it does

Usually I recommend to split it on sub items with clarification variables, to make sure that even the most junior people can understand 100% and there's almost zero chance of them misusing it. I guess we can be a bit more relaxed here, but it does add pressure for solid e2e tests 😁

Reply:
as I understand it's for admin export, hence of course it isn't mission critical either 👌

Reply:
I've added a unit test to 1) make intention of the code clearer and 2) decrease pressure for e2e tests

Reply:
And added an ananlogous test for Profile